docs: plasma: Weblate-ready tables and improved optics (docs_misc_9) (#2056)
LinuxCNC/linuxcnc: docs/src/hal/tools.adoc

refer to section  <<sec:tutorial-halscope>> in the tutorial chapter,
which explains the basics.

The halscope "File" menu selector provides buttons to save a configuration
or open a previously saved configuration.  When halscope is terminated,

> the last configuration is saved in a file named autosave.halscope.

Reviewer: ```suggestion
the last configuration is saved in a file named `autosave.halscope`.
```

[NOTE]

> `halreport` generates a connections report for a running HAL
> application to aid in designing and verifying connections.
> Pin types and current values are not shown.  For this information use
> applications like `halshow`, `halmeter`, `halscope` or the 'show'
> command available with command-line `halcmd` program.

Reviewer: ```suggestion
`halreport` generates a connections,  pin types, and current values
report for a running HAL application to aid in designing and verifying
connections. +
Use this information with applications like `halshow`, `halmeter`,
`halscope` or the  `halcmd  show` command in a terminal.
```

Author: I am not fully happy about this paragraph (also not with the original). One setence the change by @silopolis is left out - that is fine? I cannot judge myself, really. Also, can we have the report mentioned earlier? There is otherwise the danger of this sentence being missinterpreted.

Reviewer 2: Why aren't you happy with the original one?

Reviewer: > I am not fully happy about this paragraph (also not with the original). One setence the change by @silopolis is left out - that is fine? I cannot judge myself, really. Also, can we have the report mentioned earlier? There is otherwise the danger of this sentence being missinterpreted.

Realised I made a mistake in my first proposal. Edited and corrected.
About the original "For this information..." sentence, it just doesn't seem english or make sense to me, but I'm french 😅

Author: Adopted @silopolis' version but removed the "Note:" and added a sentence.
